src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/SunPKCS11.java

```diff
@@ -546,6 +549,7 @@ private static void register(Descriptor d) {
         String P11PBECipher        = "sun.security.pkcs11.P11PBECipher";
         String P11Signature        = "sun.security.pkcs11.P11Signature";
         String P11PSSSignature     = "sun.security.pkcs11.P11PSSSignature";
+        String P11KDF              = "sun.security.pkcs11.P11KDF";
```

Comment:
`P11KDF` should be `P11HKDF` with the latest renaming.

Reply:
That's right. Fixed.